blockwork/activities/bootstrap.py

```diff
@@ -15,16 +15,23 @@
 import logging
 
 import click
-
-from ..bootstrap import Bootstrap
+from ..bootstrap import Bootstrap, BwBootstrapMode
```

Comment:
Super minor - nice to separate imports from inside the project from those outside - we should setup lint to tidy this up :)